$NetBSD$

Disable ucontext.h detection, which leads to problems on some platforms.
Shell portabibility fix.
Disable univeral binary build on Mac OS X.

--- configure.orig	2013-03-14 16:58:22.000000000 +0000
+++ configure
@@ -19773,7 +19773,7 @@ fi
 
 
 
-for ac_header in limits.h unistd.h ucontext.h
+for ac_header in limits.h unistd.h
 do
 as_ac_Header=`echo "ac_cv_header_$ac_header" | $as_tr_sh`
 if eval "test \"\${$as_ac_Header+set}\" = set"; then
@@ -20524,7 +20524,7 @@ LIB_SUFFIX=""
 lib_suffix_valid="no"
 
 lib_namespace=""
-if test "x${library_namespace_versioning}" == xyes ; then
+if test "x${library_namespace_versioning}" = xyes ; then
     cat >>confdefs.h <<_ACEOF
 #define IMATH_INTERNAL_NAMESPACE Imath_${ILMBASE_VERSION_API}
 _ACEOF
@@ -20545,7 +20545,7 @@ _ACEOF
     lib_namespace="${ILMBASE_VERSION_API}"
     LIB_SUFFIX="${ILMBASE_VERSION_API}"
     lib_suffix_valid="yes"
-elif test "x${library_namespace_versioning}" == xno ; then
+elif test "x${library_namespace_versioning}" = xno ; then
     cat >>confdefs.h <<_ACEOF
 #define IMATH_INTERNAL_NAMESPACE Imath
 _ACEOF
@@ -20583,7 +20583,7 @@ _ACEOF
 fi
 
 
-if test "x${lib_suffix_valid}" == xyes ; then
+if test "x${lib_suffix_valid}" = xyes ; then
 LIB_SUFFIX_DASH="-${LIB_SUFFIX}"
 
 
@@ -20622,7 +20622,7 @@ else
   custom_public_namespace=no
 fi;
 
-if test "x${custom_public_namespace}" == xyes ; then
+if test "x${custom_public_namespace}" = xyes ; then
     { echo "$as_me:$LINENO: WARNING: Enabling 'custom user namespace' requires an additional argument, reverting to default" >&5
 echo "$as_me: WARNING: Enabling 'custom user namespace' requires an additional argument, reverting to default" >&2;}
     cat >>confdefs.h <<\_ACEOF
@@ -20640,7 +20640,7 @@ _ACEOF
 
     public_namespace=""
     custom_public_namespace=no
-elif test "x${custom_public_namespace}" == xno ; then
+elif test "x${custom_public_namespace}" = xno ; then
     cat >>confdefs.h <<\_ACEOF
 #define IMATH_NAMESPACE Imath
 _ACEOF
@@ -20690,23 +20690,7 @@ else
 fi;
 
   if test "${osx_arch}" != default ; then
-          osx_arch_flags="-arch ${osx_arch// / -arch }"
      CXXFLAGS="$CXXFLAGS $osx_arch_flags"
-
-          if test "$enable_dependency_tracking" != no ; then
-        osx_arch_array=( ${osx_arch} )
-        if test ${#osx_arch_array[@]} -gt 1 ; then
-          { { echo "$as_me:$LINENO: error: building multiple OS X architectures requires --disable-dependency-tracking.
-Please re-run configure with these options:
-  --enable-osx-arch=\"${osx_arch}\" --disable-dependency-tracking
-  		  " >&5
-echo "$as_me: error: building multiple OS X architectures requires --disable-dependency-tracking.
-Please re-run configure with these options:
-  --enable-osx-arch=\"${osx_arch}\" --disable-dependency-tracking
-  		  " >&2;}
-   { (exit 1); exit 1; }; }
-        fi
-     fi
   fi
 
     # Check whether --enable-osx-sdk or --disable-osx-sdk was given.
